elliottlib/util.py

```diff
@@ -544,21 +544,26 @@ def isolate_timestamp_in_release(release: str) -> Optional[str]:
     return None
 
 
-def get_golang_container_nvrs(nvrs, logger):
+def get_golang_container_nvrs(nvrs: List[Tuple[str, str, str]], logger) -> Dict[str, Dict[str, str]]:
+    """
+    :param nvrs: a list of tuples containing (name, version, release) in order
+    :param logger: logger
+
+    :return: a dict mapping nvr 'name' to a dict containing nvr and go-builder as values
+    """
     all_build_objs = brew.get_build_objects([
         '{}-{}-{}'.format(*n) for n in nvrs
     ])
-    go_container_nvrs = {}
+    go_nvr_map = {}
     for build in all_build_objs:
         go_version = 'N/A'
         nvr = (build['name'], build['version'], build['release'])
         name = nvr[0]
         if 'golang-builder' in name or 'go-toolset' in name:
             go_version = golang_builder_version(nvr, logger)
-            go_container_nvrs[name] = {
-                'nvr': nvr,
-                'go': go_version
-            }
+            if go_version not in go_nvr_map:
```

Comment:
go_nvr_map.keys()

Reply:
The `in` comparison for dict works as intended (unless I'm missing something)